### PR TITLE
FEATURE(CI): Add various docker images for testing purpose

### PR DESCRIPTION
**Pull request by cdelgehier**
##### SUMMARY

Image with ansible 2.9 and python3
- openio/centos7_ansible:2.9
- openio/centos8_ansible:2.9
- openio/ubuntu18.04_ansible:2.9

Images with sshd
- openio/centos7_sshd:latest
- openio/centos8_sshd:latest
- openio/ubuntu18.04_sshd:latest

 ##### IMPACT
 N/A

 ##### ADDITIONAL INFORMATION

### CODE REVIEW

File: openio-ci/ansible/centos/7/Dockerfile_2.9 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    rm -f /lib/systemd/system/multi-user.target.wants/*; \
+    rm -f /etc/systemd/system/*.wants/*; \
+    rm -f /lib/systemd/system/local-fs.target.wants/*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+    rm -f /lib/systemd/system/basic.target.wants/*; \
+    rm -f /lib/systemd/system/anaconda.target.wants/*; \
+    yum -y upgrade; \
+    yum -y install epel-release; \
+    yum -y install git sudo rsyslog python3 python3-libselinux iproute python3-netaddr rsyslog;\
```
Comment (vincent-legoll) [inline openio-ci/ansible/centos/7/Dockerfile_2.9]: double rsyslog

File: openio-ci/ansible/centos/7/Dockerfile_2.9 [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    yum clean all; \
+    : Can't log kernel messages unless we're privileged; \
+    sed -i 's/^\($ModLoad imklog\)/#\1/' /etc/rsyslog.conf; \
```
Comment (vincent-legoll) [inline openio-ci/ansible/centos/7/Dockerfile_2.9]: is this dollar matched properly ?

Comment (cdelgehier) [reply openio-ci/ansible/centos/7/Dockerfile_2.9]:
```
docker run -it openio/centos7_ansible:2.9 grep imklog /etc/rsyslog.conf
#$ModLoad imklog # reads kernel messages (the same are read from journald)
```

**Comment by vincent-legoll** [comment]
OK LGTM

**Comment by cdelgehier** [comment]
@dduportal So I have to close this PR ?

**Review by vincent-legoll — APPROVED** [review]

**Pull request merged**